app/components/Errors.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { map } from "lodash";
+import CSSModules from "react-css-modules";
+import styles from "../styles/components/errors.css";
+
+class Errors extends React.Component {
+  render () {
+    const { errors } = this.props;
+    const errorsMap = map(errors, (error, index) => {
```

Comment:
ehhhhhhh I'm not sold. Seeing too much js mixed with mark up seems a little off to me. Also the const kind of describes whats going on. What do you think of creating a property `renderErrors(errors)` or something instead?